### ISSUE TITLE
Dialogs: Cannot type `o` or `c` to input dialog due to they being registered as shortcuts

### ISSUE THREAD

**Issue by d-biehl** [robotframework/robotframework#4812]
Given this short test:
```
*** Settings ***
Library   Dialogs

*** Test Cases ***
Get Input and Verify correct answer
    ${answer}    Dialogs.Get Value From User     What's your name?
    Should Be Equal    ${answer}    Carola
```

If I type `c` to the input field on the dialog the dialog closes/cancels the dialog. 
If I type `o` to the input field the dialog closes but there is no value returned.

**Comment by pekkaklarck** [comment]
This is due to buttons getting shortcuts in RF 6.1 (#4636). They make operating dialogs more convenient without a mouse, along with other enhancements to Dialogs, but they clearly shouldn't be active when user is typing something. Need to investigate can we disable them only when the input field has focus or should they be disabled with input dialogs altogether.

**Comment by pekkaklarck** [comment]
This is a pretty severe regression. I'm on holiday, but try to get this fixed and RF 6.1.1 release relatively soon. There are few other things that need to be fixed for 6.1.1, but I hope to get it out next week.
